test/extensions/common/wasm/wasm_vm_test.cc

```diff
@@ -49,7 +49,16 @@ class BaseVmTest : public testing::Test {
   Stats::ScopeSharedPtr scope_;
 };
 
-TEST_F(BaseVmTest, NoRuntime) { EXPECT_EQ(createWasmVm(""), nullptr); }
+TEST_F(BaseVmTest, UnspecifiedRuntime) {
+  auto wasm_vm = createWasmVm("");
+  // Envoy is built with "--define wasm=disabled", so no Wasm engine is available
+  if (getFirstAvailableWasmEngineName().empty()) {
+    EXPECT_TRUE(wasm_vm.get() == nullptr);
+  } else {
+    EXPECT_TRUE(wasm_vm.get() != nullptr);
+    EXPECT_TRUE(std::string(getFirstAvailableWasmEngineName()).find(wasm_vm->getEngineName()));
```

Comment:
find() does not return true/false.

Suggest using `EXPECT_THAT(std::string(getFirstAvailableWasmEngineName()), HasSubstr(wasm_vm->getEngineName()));` if that is what you mean.

Reply:
Thanks for pointing out, will fix.